SwahiliBox/swahilibox-events-api
# Implement login

#### What does this PR do?
Implements login.

#### Description of Task to be completed?
This Pr implements this issue #23 

### How should this be manually tested?
First register a user then send `email` and `password` to  `/login` endpoint, they should then get a success message and a token.

#### Any background context you want to provide?
Currently users can signup but unable to login into the application. 

#### What are the relevant tasks or issues?
It should implement this issue #23
